rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -106,3 +120,38 @@ def test_convex_staking_balances(
         amount=FVal('378.311754894794233025'),
         usd_value=FVal('567.4676323421913495375'),
     )
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('optimism_accounts', [['0x78C13393Aee675DD7ED07ce992210750D1F5dB88']])
+def test_velodrome_v2_staking_balances(
+        optimism_inquirer: 'OptimismInquirer',
+        optimism_transaction_decoder: 'OptimismTransactionDecoder',
+        optimism_accounts: list[ChecksumEvmAddress],
+        inquirer: 'Inquirer',  # pylint: disable=unused-argument
+) -> None:
+    """Check that balances of velodrome v2 gauges are properly queried."""
+    database = optimism_transaction_decoder.database
+    tx_hex = deserialize_evm_tx_hash('0xed7e13e4941bba33edbbd70c4f48c734629fd67fe4eac43ce1bed3ef8f3da7df')  # transaction that interacts with the gauge address  # noqa: E501
+    get_decoded_events_of_transaction(  # decode events that interact with the gauge address
+        evm_inquirer=optimism_inquirer,
+        database=optimism_transaction_decoder.database,
+        tx_hash=tx_hex,
+    )
+    balances_inquirer = VelodromeBalances(
+        database=database,
+        evm_inquirer=optimism_inquirer,
+        chain=SupportedBlockchain.OPTIMISM,
+    )
+    balances = balances_inquirer.query_balances()  # queries the gauge balance of the address if the address has interacted with a known gauge  # noqa: E501
+    acc = optimism_accounts[0]
+    user_balance = balances[acc]
```

Comment:
please don't create variables that are used only in a single place.
```suggestion
    user_balance = balances[optimism_accounts[0]]
```